pkg/spaces/utils.go

```diff
@@ -83,6 +83,9 @@ func EncodeResourceID(r *provider.ResourceId) string {
 // in the format <storage_id>$base32(<path>).
 func EncodeSpaceID(storageID, path string) string {
 	encodedPath := base32.StdEncoding.EncodeToString([]byte(path))
+	if encodedPath == "" {
```

Comment:
isn't this equivalent to:
`if path == "" { ... }` ?

Reply:
yes :)